reviews/workflow.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+# TODO use model_utils.Choices, once it supports Django 1.11
+
+PRE_MODERATION = 'pre-moderation'
```

Comment:
It may be worth installing the backport of python3's enum to handle constants in a slightly nicer way.
IIRC The Django integration with enums for choices is somewhere between amazing and terrible, but not worth installing any of the additional packages for.